chainer_compiler/chainer_compiler.py

```diff
@@ -311,7 +311,7 @@ def forward(self, *args):
 
         runtime_kwargs = {}
         if (self.runtime_kwargs is not None and
-            self.num_iterations % (self.quiet_period + 1) == 0):
+           self.num_iterations % (self.quiet_period + 1) == 0):
```

Comment:
Could you revert this unnecessary change?

Reply:
It seems that `flake8` outputs an error at this part. But if we don't need to care, then I will revert.